fastify.js

```diff
@@ -358,6 +360,16 @@ function build (options) {
     return this._contentTypeParser.hasParser(contentType)
   }
 
+  function handleClientError (e, socket) {
+    socket.end(
+      JSON.stringify({
+        error: http.STATUS_CODES['400'],
+        message: 'Client Error',
+        statusCode: 400
+      })
```

Comment:
I thought about that, but I like to be consistent with the [standard error](https://github.com/fastify/fastify/blob/master/docs/Reply.md#errors) we use across Fastify:
```js
{
  error: String        // the http error message
  message: String      // the user error message
  statusCode: Number   // the http status code
}
```